### DISCUSSION TITLE
Welcome to nobuco Discussions!

### DISCUSSION THREAD

**Discussion by AlexanderLutsenko** [AlexanderLutsenko/nobuco#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dazzle-me** [comment]
Hi! I'm an active Kaggler and DL engineer, amazing tool, very intuitive, has good debug UI and... works! 
Just wanted to say that I made a post on [Kaggle forum](https://www.kaggle.com/competitions/asl-signs/discussion/402545) about nobuco.

Kaggle now hosts the competition about sign language recognition and the submission format is .tflite model, however most of the people use pytorch for training their models (just like me), and they desperately in need of tool like nobuco to convert their models to Keras and after to TFLite. I hope this post will find attraction and this library receive more attention.
